src/elements/checkbox/checkbox-panel/checkbox-panel.ts

```diff
@@ -16,11 +16,16 @@ import type {
   SbbStateChange,
 } from '../../core/interfaces/types.js';
 import { panelCommonStyle, SbbPanelMixin, SbbUpdateSchedulerMixin } from '../../core/mixins.js';
-import { checkboxCommonStyle, SbbCheckboxCommonElementMixin } from '../common.js';
+import {
+  checkboxCommonStyle,
+  SbbCheckboxCommonElementMixin,
+  type SbbCheckboxSize,
+} from '../common.js';
 
 import '../../screen-reader-only.js';
 import '../../visual-checkbox.js';
 
+export type SbbCheckboxPanelSize = Omit<SbbCheckboxSize, 'xs'>;
```

Comment:
```suggestion
export type SbbCheckboxPanelSize = 's' | 'm';
```
Nitpick but I believe that extension by exclusion is generally a bad pattern